### PR TITLE
Create 0x302f66fAF69e6c0c20aBA767a3F7e13276E50c7D

### PR DESCRIPTION
**Pull request by Samino86**

### CODE REVIEW

File: 0x302f66fAF69e6c0c20aBA767a3F7e13276E50c7D [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
```
Comment (omidsfd) [inline 0x302f66fAF69e6c0c20aBA767a3F7e13276E50c7D:11]: G

**Review by 11dest — APPROVED** [review]
Yes this is looking good

File: 0x302f66fAF69e6c0c20aBA767a3F7e13276E50c7D (hunk 1 above)
Comment (Darren0212) [reply 0x302f66fAF69e6c0c20aBA767a3F7e13276E50c7D:11]: scam project!!!

**Comment by YOUSU198** [comment]
F,.

File: 0x302f66fAF69e6c0c20aBA767a3F7e13276E50c7D (hunk 1 above)
Comment (YOUSU198) [reply 0x302f66fAF69e6c0c20aBA767a3F7e13276E50c7D:11]: TF

**Review by YOUSU198 — COMMENTED** [review]
GDRF

**Pull request closed without merging**